desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/steps/seller/SellerStep3View.java

```diff
@@ -157,13 +154,19 @@ public void activate() {
             }
         });
 
-        // we listen for updates on the trade autoConfirmResult field
-        if (assetTxProofResultField != null) {
-            trade.getAssetTxProofResultUpdateProperty().addListener(proofResultListener);
+        if (useXmrTxProof) {
+            proofResultListener = (observable, oldValue, newValue) -> {
+                applyAssetTxProofResult(trade.getAssetTxProofResult());
+            };
+
+            // we listen for updates on the trade autoConfirmResult field
+            if (assetTxProofResultField != null) {
+                trade.getAssetTxProofResultUpdateProperty().addListener(proofResultListener);
+                applyAssetTxProofResult(trade.getAssetTxProofResult());
```

Comment:
```suggestion
```

Reply:
There is more to clean up. See new commit.